.github/workflows/nightly.yaml

```diff
@@ -6,11 +6,13 @@ on:
 
```

Comment:
Also add the following so we can trigger this (but revert it later)

on:
  push:
    branches:
      - main
  pull_request:
    branches:
      - main

Reply:
Thanks for the suggestion! I do see that this file has a `workflow_dispatch:`. From what I gather, this should ideally allow us to trigger it manually as soon as we merge so we may not need the `on: push:`.

Meanwhile, I will add a `pull_request` section so that it runs in the PR.

Reply:
Added pull_request to the commit. Once the result is available from the run, will remove it again.

Reply:
The nightly job is green now! :)
I'll be removing it in the next push.

![image](https://github.com/project-zot/zot/assets/30438425/ba6531ab-b941-4443-90f4-28eb5f0a6b5b)

Reply:
This has now been removed.